public/scripts/extensions/connection-manager/edit.html

```diff
@@ -0,0 +1,16 @@
+<div>
+    <h3>Included settings:</h3>
+    <div class="justifyLeft flex-container flexFlowColumn flexNoGap">
+        {{#each settings}}
+        <label class="checkbox_label">
+            {{#if this}}
+                <input type="checkbox" value="{{@key}}" name="exclude" checked>
+            {{else}}
+                <input type="checkbox" value="{{@key}}" name="exclude">
+            {{/if}}
```

Comment:
Could've been one line. But that's just me being nitpicky again.
At least I think that's how this templating works
```suggestion
            <input type="checkbox" value="{{@key}}" name="exclude"{{#if this}} checked{{/if}}>
```

Reply:
This works, but doesn't look as good in the syntax highlight, eh.